assets/js/modules/analytics/datastore/tags.js

```diff
@@ -260,25 +319,41 @@ export const selectors = {
 	/**
 	 * Get an existing tag on the site, if present.
 	 *
-	 * Returns an object with the shape when successful:
-	 * ```
-	 * {
-	 *   accountID = null,
-	 *   propertyID = null,
-	 * }
-	 * ```
-	 *
 	 * @since n.e.x.t
 	 *
 	 * @param {Object} state Data store's state.
-	 * @return {?Object} Site connection info.
+	 * @return {?string} Existing tag, or `null` if none.
+	 *                   Returns `undefined` if not resolved yet.
 	 */
 	getExistingTag( state ) {
 		const { existingTag } = state;
 
 		return existingTag;
 	},
 
+	/**
+	 * Checks whther the user has access to the existing Analytics tag.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return {?boolean} true or false if tag permission is available,
```

Comment:
The `?` prefix means the parameter is nullable so it essentially means `boolean` or `null`. 

I don't think JSdoc considers `null` a type of its own - it isn't mentioned in the docs; nor is `undefined` for that matter so I think `undefined` and `null` are interchangable as unprovided values.

![image](https://user-images.githubusercontent.com/1621608/79576107-72be1e00-80cb-11ea-8516-c2a94c1e4161.png)
https://jsdoc.app/tags-type.html

Strictly speaking, `null` is actually an Object so it would be weird to document it as `{(boolean|Object)}`

![image](https://user-images.githubusercontent.com/1621608/79576340-ca5c8980-80cb-11ea-878f-f475b740f220.png)

To account for `undefined` as a valid parameter, we should use the optional syntax:
```
@param {number} [foo]
```

That doesn't apply to a `@return` type though so, maybe this should be `({boolean|undefined})`

Thoughts? 😄